packages/php/email-editor/src/Engine/Templates/class-templates-registry.php

```diff
@@ -37,11 +37,6 @@ public function initialize(): void {
 	 * @return void
 	 */
 	public function register( Template $template ): void {
-		// The function was added in WordPress 6.7. We can remove this check after we drop support for WordPress 6.6.
-		if ( ! function_exists( 'register_block_template' ) ) {
-			return;
-		}
-
```

Comment:
A minimum version bump is fine with the email editor, but please don't remove this check. We plan to publish the editor as a stand-alone library and want this check to remain in place for now.

Reply:
@triple0t so do you plan to continue supporting WP 6.6 in the standalone library?  It sounds like by leaving this check in place you do anticipate supporting some level of sites with WP 6.6.

Reply:
@joshuatf  Feedback: p1747997572533849/1747994921.205199-slack-C0313FPUG74
We will be following L-1 for this as well.